jsapp/js/components/permissions/utils.ts

```diff
@@ -460,6 +461,66 @@ export function getPermLabel(perm: PermissionResponse) {
   return '???'
 }
 
+/**
+ * Returns the permission label with the correct suffix depending on the given asset type.
+ *
+ * Example: if we are sharing a library collection, the permissions will all say "[View, Edit, Manage] collection"
+ */
+export function getFriendlyPermLabelWithSuffix(
+  assetType: AssetTypeName | undefined,
+  perm?: PermissionResponse,
+  checkboxNameParam?: CheckboxNameAll,
```

Comment:
With the proposition in comments above, I think we could only use `assetType` and `perm` here, right? Because all the places that will use this function would pass `PermissionResponse`.

Reply:
Commented on the propositions above, but they still yielded simpler paramters :)